app/src/main/java/ar/com/wolox/lucasdelatorre/training/instances/NewsInstance.java

```diff
@@ -1,84 +1,106 @@
 package ar.com.wolox.lucasdelatorre.training.instances;
 
-import java.util.ArrayList;
-import java.util.Date;
+import ar.com.wolox.lucasdelatorre.training.Utils;
 
 public class NewsInstance {
 
-    private String mTitle;
-    private String mDescription;
-    private Date mTime;
-    private boolean mLike;
+    private String objectId;
+    private String id;
+    private String[] likes;
+    private boolean like;
+    private String picture;
+    private String text;
+    private String title;
+    private String createdAt;
+    private String updatedAt;
+    private String diffTime;
+
+    public void initialize(String reader) {
+        this.like = false;
+
+        for (String like : likes) {
```

Comment:
Si te referís al for, es para recorrer todos los likes que tiene la noticia y ver si dentro de ese array aparece el id del reader (o sea, del usuario logueado). Si ese es el caso, la noticia debería tener un like y aparecer con un corazón coloreado en la lista.

El initialize es simplemente para que no haya que hacer ese procesamiento cada vez que se llame a la vista.